### PR TITLE
Add descriptions for new output fields

### PR DESCRIPTION
**Pull request by jsf9k**
cc @echudow 

I took a stab at adding descriptions for the new output fields in the README.  Please review and let me know (or commit) if anything is not accurate.

### CODE REVIEW

**Comment by jsf9k** [comment]
This pull request **fixes 1 alert** when merging e656fd22c93ff000a69a7ac6aa0f91b329e232e7 into cd5d0c914deb2310cac79d8a8cc2e3d4438240b2 - [view on LGTM.com](https://lgtm.com/projects/g/cisagov/pshtt/rev/pr-1b060bccb419b0a9eac36008e58030f6f3e00261)

**fixed alerts:**

* 1 for Nested loops with same variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: README.md [hunk 1]
```diff
@@ -128,6 +128,14 @@ The following values are returned in `results.csv`:
 * `Canonical URL` - One of the four endpoints described above; a
   judgment call based on the observed redirect logic of the domain.
 * `Live` - The domain is "live" if any endpoint is live.
+* `HTTPS Live` - The domain is "HTTPS live" if any HTTPS endpoint is
+  live.
+* `HTTPS Full Connection` - The domain is "fully connected" if any
+  HTTPS endpoint is fully connected.  A "fully connected" HTTPS
+  endpoint is one with which pshtt could make a full TLS connection.
+* `HTTPS Client Auth Required` - A domsin requires client
```
Comment (dav3r) [inline README.md]: Typo: `domsin`

**Review by dav3r — APPROVED** [review]
LGTM except for the small typo that I noted.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: Fixed in 7054602.

**Comment by jsf9k** [comment]
This pull request **fixes 1 alert** when merging 70546028fbfd9b10a52980c33328951fc4278d02 into cd5d0c914deb2310cac79d8a8cc2e3d4438240b2 - [view on LGTM.com](https://lgtm.com/projects/g/cisagov/pshtt/rev/pr-f3ad831e04dda8c4651de535e4dacedb2930ecad)

**fixed alerts:**

* 1 for Nested loops with same variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by jsf9k** [comment]
This pull request **fixes 1 alert** when merging d7b616575882cb55fd4b684da3f7e1cc8d23e189 into cd5d0c914deb2310cac79d8a8cc2e3d4438240b2 - [view on LGTM.com](https://lgtm.com/projects/g/cisagov/pshtt/rev/pr-3b3b674ab49d9c54c50827f395d233065648d1aa)

**fixed alerts:**

* 1 for Nested loops with same variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**